Crypto/Sources/KeyPair/KeyPair.swift

```diff
@@ -110,4 +110,32 @@ public struct KeyPair: KeyPairType {
                        keySize: keySize,
                        algorithm: algorithm)
     }
+
+    public func store(tag: String) throws {
+
+        let privateSecKeyData = try privateKey.asData()
+        let publicSecKeyData = try publicKey.asData()
+
+        let addPrivateKeyQuery = [kSecClass as String: kSecClassKey,
+                                  kSecAttrKeyType as String: kSecAttrKeyTypeRSA,
+                                  kSecAttrKeyClass as String: kSecAttrKeyClassPrivate,
+                                  kSecValueData as String: privateSecKeyData,
+                                  kSecAttrApplicationTag as String: tag] as [String: Any]
```

Comment:
`[String : Any]` is that a mapping from String to Any or the opposite?

Reply:
its a dictionary (hashmap) with string as key and any type as value. Casting necessary because of objc interop